### ISSUE TITLE
Update plugin minimum versions in pubspec.yaml

### ISSUE THREAD

**Issue by LarkAscending** [flutter/friendlychat-steps#41]
The sample pubspec.yaml file in the [full_steps/step_1 folder](https://github.com/flutter/friendlychat-steps/blob/master/full_steps/step_1_set_up_firebase/pubspec.yaml) needs updating to point to the latest plugin versions:
```
dependencies:
  flutter:
    sdk: flutter
  image_picker: 0.1.0 -> 0.1.1
  google_sign_in: 0.3.1
  firebase_analytics: 0.0.4
  firebase_auth: 0.1.1 -> 0.2.0
  firebase_database: 0.0.12
  firebase_storage: 0.0.5
```
[Step 5 of the Firebase Codelab](https://codelabs.developers.google.com/codelabs/flutter-firebase/index.html#4) has the updated versions. The sample config file in this repo needs updating to match the codelab.

/cc @collinjackson @efortuna @emshack

**Comment by Fandekasp** [comment]
google_sign_in is v1.0.1, firebase_analytics v0.1.1, firebase_auth v0.3.1, firebase_database 0.0.12 and firebase_storage v0.0.7.

Using the latest packages worked until Step 8, and the code broke with Step 9 (Enable data syncing). Couldn't figure out what was wrong, eg with the line `itemBuilder: (_, DataSnapshot snapshot, Animation<double> animation)`.

A next step showing how to upgrade and debug this would be wonderful.

**Comment by LarkAscending** [comment]
@Fandekasp Your pubspec.yaml should be pointing to the versions in the original issue description (above). These are the latest combinations that have been verified to work. Let us know if you are still seeing the issue at Step 9.

And, if you could include any error messages as well as flutter doctor output, that would be totally helpful too.

**Comment by Fandekasp** [comment]
The versions you listed work fine.

**Comment by LarkAscending** [comment]
Resolved via #49
